engine/e2e-test/test_cli_engine_uninstall.py

```diff
@@ -18,7 +18,7 @@ def setup_and_teardown(self):
 
     def test_engines_uninstall_llamacpp_should_be_successfully(self):
         exit_code, output, error = run(
-            "Uninstall engine", ["engines", "uninstall", "cortex.llamacpp"]
+            "Uninstall engine", ["engines", "uninstall", "cortex.llamacpp"], timeout = None
```

Comment:
Uninstall should be quick. I don't think we should put None here. Are you experiencing a slow when uninstall engine?